### PR TITLE
🏗 Remove Single Pass Pipeline

### PR DESCRIPTION
**Pull request by kristoferbaxter**
We've been unsuccessful in shipping Single Pass Compilation for ~2 years, and the technical debt of maintaining several compilation pipelines is starting to show in PRs.

This PR removes the codebase and pipeline support for Single Pass Compilation, letting us reset and find another pathway to achieve the same goals.

### CODE REVIEW

File: build-system/compile/single-pass.js [hunk 1]
```diff
@@ -1,790 +0,0 @@
...
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-const babel = require('@babel/core');
-const babelify = require('babelify');
-const browserify = require('browserify');
-const colors = require('ansi-colors');
-const del = require('del');
-const devnull = require('dev-null');
```
Comment (rsimha) [inline build-system/compile/single-pass.js:22]: This can be removed from `package.json`.

File: build-system/compile/single-pass.js [hunk 2]
```diff
@@ -1,790 +0,0 @@
...
-const fs = require('fs-extra');
-const gulp = require('gulp');
-const gulpIf = require('gulp-if');
-const log = require('fancy-log');
-const MagicString = require('magic-string');
```
Comment (rsimha) [inline build-system/compile/single-pass.js:27]: This too can be removed from package.json.

File: build-system/compile/single-pass.js [hunk 3]
```diff
@@ -1,790 +0,0 @@
...
-const minimist = require('minimist');
-const path = require('path');
-const pkgUp = require('pkg-up');
```
Comment (rsimha) [inline build-system/compile/single-pass.js:30]: And this.

File: build-system/compile/single-pass.js [hunk 4]
```diff
@@ -1,790 +0,0 @@
-/**
```
Comment (rsimha) [inline build-system/compile/single-pass.js:1]: Maintaining this file was quite a tedious task. Thanks for cleaning it up!

Comment (kristoferbaxter) [reply build-system/compile/single-pass.js:1]: Happy to help.

**Pull request merged**